app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -64,4 +66,186 @@ class SmokeTest {
+            }.openThreeDotMenu {
+            }.sharePage {
+                // verifyShareAppsLayout()
+                // verifyShareOverlay()
+            }.closeShareDialogReturnToPage {
+            }.openThreeDotMenu {
+            }.refreshPage {
+                // verifyPageContent(defaultWebPage.content)
+            }
+        }
+    }
+
+        @Test
+        fun verifyPageMainMenuItemsListInPortraitPrivateMode() {
+            val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+            homeScreen {
+                togglePrivateBrowsingModeOnOff()
+                navigationToolbar {
+                    navigationToolbar {
+                    }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+                    }.openThreeDotMenu {
+                        verifyThreeDotMainMenuItems()
+                    }.openHistory {
+                        verifyEmptyHistoryView()
+                    }.goBackToBrowser {
+                    }.openThreeDotMenu {
+                    }.openBookmarks {
+                        verifyBookmarksMenuView()
+                        verifyEmptyBookmarksList()
+                    }.goBackToBrowser {
+                    }.openThreeDotMenu {
+                    }.openSettings {
+                        verifySettingsView()
+                    }.goBackToBrowser {
+                    }.openThreeDotMenu {
+                    }.openFindInPage {
+                        verifyFindInPageSearchBarItems()
+                    }.closeFindInPage {
+                    }.openThreeDotMenu {
+                    }.addToFirefoxHome {
+                        verifySnackBarText("Added to top sites!")
+                    }.openTabDrawer {
+                    }.openHomeScreen {
+                        togglePrivateBrowsingModeOnOff()
+                        verifyExistingTopSitesTabs(defaultWebPage.title)
+                        togglePrivateBrowsingModeOnOff()
+                    }.openTabDrawer {
+                    }.openTab(defaultWebPage.title) {
+                    }.openThreeDotMenu {
+                    }.openAddToHomeScreen {
+                        verifyShortcutNameField(defaultWebPage.title)
+                        clickAddShortcutButton()
+                        clickAddAutomaticallyButton()
+                        verifyShortcutIcon()
+                    }.openHomeScreenShortcut(defaultWebPage.title) {
+                    }.openThreeDotMenu {
+                    }.bookmarkPage {
+                        verifySnackBarText("Bookmark saved!")
+                    }.openThreeDotMenu {
+                    }.sharePage {
+                        // verifyShareAppsLayout()
+                        // verifyShareOverlay()
+                    }.closeShareDialogReturnToPage {
+                    }.openThreeDotMenu {
+                    }.refreshPage {
+                        // verifyPageContent(defaultWebPage.content)

```

Comment:
Could you check the text in the url bar after the refresh? thinking about different ways to do the checks and enable these options...

Reply:
Okay